config/config.go

```diff
@@ -155,6 +155,7 @@ func init() {
 
 	cfg.SetDefault("ovs.ovsdb", "unix:///var/run/openvswitch/db.sock")
 	cfg.SetDefault("ovs.oflow.enable", false)
+	cfg.SetDefault("ovs.oflow.openflow_versions", "OpenFlow10,OpenFlow13")
```

Comment:
Yes but ultimately it is just an argument given to command and you will have to join back your elements. It will not make the definition more flexible. So I do not really see the point.

Reply:
Only for consistency. As we can express lists in YAML, why use a comma delimited string ?

Reply:
Updated with an array of strings.